src/test/resources/application-connectedtest.properties

```diff
@@ -1,6 +1,7 @@
 # ct for connected test
 ct.ingestBucket=jade-testdata
 ct.ingestRequesterPaysBucket=jade_testbucket_requester_pays
+ct.googleBillingAccountId=00708C-45D19D-27AAFA
```

Comment:
How does the connected test know to use this property? The integration test properties uses `it` instead. Does it skip the `ct` prefix?

Reply:
There are specific configuration objects used by the different tests (not really automatic but by convention...you could theoretically access access integration test config from a connected test and vice-versa...just thought maybe it made sense to keep them separate). If you look at `bio.terra.app.configuration.ConnectedTestConfiguration` it accesses all properties that are prefixed with `ct`. That's how it knows to read in there.  We then get a an instance of that bean in the connected test

Reply:
This is very helpful. Thank you!